src/main/java/uk/gov/hmcts/reform/et/helper/mapping/ServiceHearingValuesMapping.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.et.helper.mapping;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.hmcts.et.common.model.ccd.CaseDetails;
+import uk.gov.hmcts.reform.et.model.service.hearingvalues.ServiceHearingValues;
+
+
+@Slf4j
+public final class ServiceHearingValuesMapping {
+
+    private ServiceHearingValuesMapping() {
+    }
+
+    public static ServiceHearingValues mapServiceHearingValues(CaseDetails caseDetails) {
+        log.info("Mapping hearing values for Case id : {}, for use in generating Service Hearing Values",
+                 caseDetails.getCaseId());
+        return ServiceHearingValues.builder()
```

Comment:
Is it intended to leave all fields as null/false?

Reply:
Yeah the intention was to just setup the controller without CCD integration. The next ticket https://tools.hmcts.net/jira/browse/RET-3636 adds CCD and creates the services hearing values based on the case details from CCD.